### ISSUE TITLE
Staging Review finding: "Learn more" buttons are not descriptive

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#126944]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** govcio-vebt
**Product:** Comparison Tool
**Feature:** Student feedback grid

## Findings details
**VA.gov Experience Standard - issue:** [User is uncertain where a button or link leads.](https://design.va.gov/about/experience-standards/#user-knows-where-a-button-or-link-leads)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility), @kkitagawa-bosch (content)

### Description

"Learn more" buttons on the institution page are not descriptive. Ensuring buttons provide context for screen reader users is important as users  tab through the experience, they can be aware of what each button does without having to relay on additional context clues . Button text, when repeated across an experience, can be confusing due to the lack of context. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="962" height="208" alt="learn more" src="https://github.com/user-attachments/assets/555030a5-a726-434b-9dc8-fd5a36e85507" />

</details>

## Recommended action

Using the Additional information pattern may be a better approach to providing these small amounts of information:
https://design.va.gov/components/additional-info

Or, provide more context to the buttons using an `aria-label`, `aria-described-by`, or screen reader only text.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 4.1.2 Name, Role, Value - Level A](https://www.w3.org/TR/WCAG22/#name-role-value)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** [Links](https://design.va.gov/content-style-guide/links)
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
